resources/js/Pages/Articles/Show.tsx

```diff
@@ -1,8 +1,27 @@
+import { Heading } from "@/Components/Heading";
+import { Img } from "@/Components/Image";
 import { DefaultLayout } from "@/Layouts/DefaultLayout";
 
 const ArticlesShow = ({ article }: { article: App.Data.Articles.ArticleData }): JSX.Element => (
     <DefaultLayout>
-        <div className="mx-6 sm:mx-8 2xl:mx-0">{article.title}</div>
+        <article className="article-wrapper -mt-6 pb-6 sm:-mt-8  lg:mx-8 lg:mt-0 lg:overflow-hidden lg:rounded-xl lg:border lg:border-theme-secondary-300">
+            <div className="bg-white/20 px-8 pt-8 md:px-24">
```

Comment:
@alfonsobries this still has an issue with padding. Design is 1376px wide, while the implementation has padding which brings it down to 1310px.

Reply:
the content should also be max 1000px wide on desktop